app/views/users/show.html.erb

```diff
@@ -7,5 +7,5 @@
       <p><%= post.text %><p>
     </div>
   <% end %>
- 
+ <a href="<%= user_posts %>" class="btn btn-primary justify-content-center mt-3"> See all post</a>
 </div>
```

Comment:
- [ ] Clicking on each user should lead to the user show page, kindly fix this.

Before 


screenshot
![image](https://github.com/francksefu/blog_web/assets/81932478/d70b055c-3fb9-4b6c-a2ec-9dc2a841cdb5)


As I am not able to see if other requirements have been met due to this error kindly ensure you meet all requirements